core/main.go

```diff
@@ -686,12 +712,21 @@ func (t *Textile) SetLogLevels(logLevels map[string]string) error {
 
 	for key, value := range logLevels {
 		if err := logging.SetLogLevel(key, value); err != nil {
-			log.Errorf("error: %s (%s)", err, key)
-			return err
+			return nil, err
 		}
 	}
-	t.writer = writer
-	return nil
+	return writer, nil
+}
+
+// getTextileDebugLevels returns a map of textile's logging subsystems set to debug
+func getTextileDebugLevels() map[string]string {
```

Comment:
Useful with the `--debug` flag (which controls a bool on the `InitConfig`, and `RunConfig`)